lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -7,11 +7,21 @@ module Pipeline
     module Magento
       module Product
         class TopLevelAttributes < Pipeline::Stage
+          MAX_NUM_OF_OPTIONS_ACCEPTED = 3
+
           def convert(hash, record)
+            if has_too_many_options?(hash)
```

Comment:
suggest `too_many_options?` since i dont think `has` is offering any extra info here